cmd/orderimports/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - dims
+  - neolit123
+approvers:
+  - dims
+  - neolit123
```

Comment:
my previous comment on the topic of OWNERS also implied that i'm not sure who should own this.

perhaps the bigger question is - do we want to enable this linter for the whole k/k?
it would make the code nice and consistent at least.

/area code-organization

Reply:
maybe we can get comments on that if we log the issue, as mentioned here:
https://github.com/kubernetes/kubernetes/pull/103981#discussion_r681475351

Reply:
once https://github.com/kubernetes/org/issues/2859 merges you can add yourself too, @xinydev 
my take is that this initial linter PR can merge with only the 3 of us as OWNERS, others can join later if they want to.